OSCAL_REFERENCE.md

```diff
@@ -0,0 +1,56 @@
+# OSCAL Reference Doc
+
+A quick cheat-sheet of relevant fields within the SSP. This is not exhaustive and some SSPPs will likely need other fields that are not included in this document.
+
+## SSP
+
+The [SSP model](https://pages.nist.gov/OSCAL/resources/concepts/layer/implementation/ssp/) is where all information for the eventual System Security & Privacy Plan is stored.
+
+For docker-trestle, the `control-implementation` section is maintained and edited within the `control-statements` markdown files. The other sections are edited in JSON files within `system-security-plans/SYSTEM_NAME/system-security-plan/`
+
+### Metadata
+
+`metadata.json`
+
+> Metadata syntax is identical and required in all OSCAL models. It includes information such as the document's title, publication version, publication date, and OSCAL version. Metadata is also used to define roles, parties (people, teams and organizations), and locations.
+
+| Field | Description |
+| ----- | ----------- |
+| `title` | Title of the SSPP document |
+| `last-modified` | Do not edit - automatically generated timestamp |
+| `version` | Version of the SSP document. Revise whenever submitting final doc for approvals |
+| `oscal-version` | Version of OSCAL in use. trestle currently supports `1.0.4` |
+
+### System Characteristics
+
+`system-characteristics.json`
```

Comment:
Getting this capability into place is great! This is an area we will probably need to invest in with samples and tooling to be as approachable as the markdown controls.